projects/packages/forms/src/contact-form/class-contact-form.php

```diff
@@ -1356,6 +1368,7 @@ public static function parse_contact_field( $attributes, $content, $block = null
 			return $html;
 		}
 
+		// What does this actually means? What is the case where this is used?
```

Comment:
I think this was used when we were parsing the post_content and to get the form from the content. But also to pass the form object to the fields object. Since it is used in the Contact_Form_Field constructor. 

This creates a bit of a circular dependency.